src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  // readonly payApiUrl: string
+
+  async getErrorObj (errCode) {
+    const fetchUrl = this.payApi + 'codes/errors/' + errCode
+    const errObj = await axios.get(fetchUrl).catch()
+    if (errObj && errObj.data) {
+      return errObj.data
+    }
+    return null
+  }
+
+  getErrorCode (error) {
+    if (error.response?.data?.errors) {
```

Comment:
Code looks good but what if `error` itself is null?

Reply:
Looks like nothing is returned by this method if this statement is false.

Actually, if you move the optional chaining inside line 19 (const msgCode...) then you probably don't need the "if" and it will all work out.

Reply:
done

Reply:
Check for null `error` is now implemented but method still has no return value if check fails.

Reply:
the second return statement should cover that case, no?

Reply:
The second return statement is inside "if (error?...)" block. So if this "if" fails, you don't return anything.

Reply:
oh shoot I was looking at the other method (getErrorObj)...similar names, sorry. Will fix!